src/RenderItems.js

```diff
@@ -1,13 +1,51 @@
 import React, { Component } from 'react'
-import Data from './Data.js'
+// import Data from 'Data.js'
 import Item from './Item.js'
 
+
+
 export default class RenderItems extends Component {
+    state = {
+        filter: ''
+    }
+
+    keySet = [...new Set(this.props.Data.map(item => item.keyword))]
+    hornSet = [...new Set(this.props.Data.map(item => item.horns))]
+
+
+    handleChange = e => {
+        this.setState({
+            filter: e.target.value,
+
+        });
+    }
     render() {
+
+        const filteredAnimals = this.props.Data.filter((animal) => {
+            if (!this.state.filter) return true;
+
+
+            if (animal.keyword === this.state.filter || animal.horns === +this.state.filter) return true;
+
+
+
+            return false
+        })
         return (
+
             <>
-                <p>
-                    <select>
+                <h1>
+                    <select onChange={this.handleChange}>
+                        <option value=''>Show All</option>
+                        {this.keySet.map(option =>
```

Comment:
Could be written as:
  {
          this.keySet
          .map(option => {
          return <option value={option}>{option}</option>
          )}
}

This would be my preference and it's based solely on aesthetics/how it reads. I'm just showing in case you see people writing code like this.